R/deprecated.R

```diff
@@ -339,8 +339,9 @@ get_join_keys <- function(data) {
 }
 
 #' @rdname col_labels
-#' @inheritParams col_labels
-#' @param data a data.frame object
+#' @param data a `data.frame` object
+#' @param fill (`logical(1)`) boolean in case the `label` attribute does not exist if
+#'   `TRUE` the variable names is returned, otherwise `NA`.Default is `FALSE`.
```

Comment:
Can you please add these new deprecated functions in the `_pkgdown.yml` section "Deprecated functions and methods"?
Additionally, the function doc for the deprecated function `get_join_keys` has a missing object name.

Reply:
done thanks [82d8b30](https://github.com/insightsengineering/teal.data/pull/240/commits/82d8b3071cbd0793fb66cdf4d5fc844e3ad728b3)

Reply:
`get_join_keys` needs to be added to the pkgdown. There is a comment below for this so this can stay resolved.